tools/go.mod

```diff
@@ -0,0 +1,15 @@
+module github.com/moby/moby/tools
+
+go 1.17
+
+require (
+	github.com/containerd/protobuild v0.1.1-0.20211025221430-7e5ee24bc1f7
+	github.com/goware/modvendor v0.5.0
```

Comment:
Looks like this repository may have issues with licensing; the README mentions `MIT`, but the repo has no LICENSE file, so "technically" it's not licensed (GitHub itself also doesn't detect the license; other tools may also flag it); https://github.com/goware/modvendor/tree/v0.5.0


Doing a quick check on how to apply the MIT license, it looks like it requires a LICENSE or LICENSE.txt file to be in there; https://choosealicense.com/licenses/mit/

Reply:
Are we actually using this tool? (I didn't see `modvendor` being called, or is it used indirectly somehow?)

Reply:
Ah, I think I forgot to remove it completely. I tried it and it didn't work. cc @crazy-max re licensing.

Reply:
I've dropped it now!